### PR TITLE
created TC-187, added method imageUpload, added selector imageStoryPa…

### PR DESCRIPTION
**Pull request by isenkovska**
created TC-187, added method imageUpload(can be used for image field tests), added selector imageStoryPage, added image test data to testData.

### CODE REVIEW

File: data/selectors.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ const elements = {
     "image": "span.ant-upload",
     "imgAlt": "img[alt]",
     "submit": ".ant-btn",
+    "imageStoryPage": "img",
```
Comment (EAGauss) [inline data/selectors.js:29]: It's better to use a more specific selector to check if user was redirected to the story page:
`"imageStoryPage": "img[alt='hero_picture']",`

Comment (EAGauss) [inline data/selectors.js:29]: It's better to use a more specific selector to check if user was redirected to the story page:
`"imageStoryPage": "img[alt='hero_picture']",`

File: test/extended/story.js [hunk 2]
```diff
@@ -65,4 +65,13 @@ describe('Story field', function () {
         expect(countGender).toEqual(exp.genderCount);
     });
 
+    it.only('TC-187 `Image is appearing at the top right corner of the story.', function () {
```
Comment (EAGauss) [inline test/extended/story.js:68]: The quote ` opens but doesn't closes.

**Review by EAGauss — CHANGES_REQUESTED** [review]
The code itself is Okay but you should make some changes.

File: test/extended/story.js (hunk 2 above)
Comment (EAGauss) [reply test/extended/story.js:68]: And yeah, you pushed "it.only" to the main branch.

**Review by irinadash — APPROVED** [review]

**Review by irinadash — APPROVED** [review]

**Pull request merged**